File: WinForms/VB/SpreadsheetMainDemo/Modules/Outline.vb
Imports DevExpress.XtraBars.Ribbon

Namespace DevExpress.XtraSpreadsheet.Demos

    Public Partial Class OutlineModule
        Inherits SpreadsheetRibbonTutorialControlBase

        Public Sub New()
            InitializeComponent()
            spreadsheetControl1.LoadDocument(DemoUtils.GetRelativePath("OutlineGrouping_template.xlsx"))
        End Sub

        Protected Friend Overrides ReadOnly Property SelectedRibbonPage As RibbonPage
            Get
                Return dataRibbonPage1
            End Get
        End Property
    End Class
End Namespace

File: WinForms/CS/SpreadsheetMainDemo/Modules/Outline.cs
using DevExpress.XtraBars.Ribbon;

namespace DevExpress.XtraSpreadsheet.Demos {
    public partial class OutlineModule : SpreadsheetRibbonTutorialControlBase {
        public OutlineModule() {
            InitializeComponent();
            spreadsheetControl1.LoadDocument(DemoUtils.GetRelativePath("OutlineGrouping_template.xlsx"));
        }

        protected internal override RibbonPage SelectedRibbonPage {
            get {
                return dataRibbonPage1;
            }
        }
    }
}
